README.md

```diff
@@ -1,3 +1,36 @@
+S3L1M created a very useful program to help revert files captured by `ground.exe` virus.  I got infected, amd noticed it fairly quickly when I noticed `ground.exe` running in task manager.  I would like to create this updated readme with more detailed instructions for using Ground-Damage-Fixer. Of course, you already have `Ground` stopped and are stuck with this messy aftermath. I included how I removed `Ground.exe` below in case you came across this page while still searching for a way to remove the virus.
+
+The command to manualy unhide and retsore permissions to a file/folder is
+```
+attrib -H -S *.exe /S /D
+```
+
+1) Run `Super Unhider`
+2) Click on `Add Files`
+2) search for files with `size:534,016`
+![Alt Text](https://github.com/bizzar721/Ground-Damage-Fixer/blob/master/1111.png)
+3 )After search is complete, use `CTRL-A` to select all, then `CTRL-X` to cut (Icons should all now be transparent)
+![Alt Text](https://github.com/bizzar721/Ground-Damage-Fixer/blob/master/2222.png)
+4) Click `Open`
+5) Open `Recycle Bin`, then `CTRL-V` to paste the infected `.exe`s
+![Alt Text](https://github.com/bizzar721/Ground-Damage-Fixer/blob/master/3333.png)
+6) Click on the top entry in `Super Unhider` and click `Unhide` (runs from selected downward)
+7) Give it a minute to finish, then select the top entry again and click `Rename`
+
+All files should now be restored.
+
+# Removing Ground.exe
+* I recomend having Malwarebytes Anti-malware installed (free)
+* Boot into   `Safe Mode` . This can be done by typing `msconfig` in the search bar.
```

Comment:
I create that program 7 years ago. So, I don't remember all the details of what I did baack then. It is better to remove the program for the startup program list (using any utility or manaually by _regedit_) after removing the virus itself.